src/builtin/callRecording.ts

```diff
@@ -0,0 +1,214 @@
+            message.content.length
+          )
+          let updatedCallId = callId.split('#')[0]
+          if (updatedCallId[0] === '/') {
+            updatedCallId = updatedCallId.substring(1, updatedCallId.length)
+          }
+          if (
+            tribe.callRecording === 1 &&
+            tribe.jitsiServer.length !== 0 &&
+            tribe.jitsiServer === jitsiServer &&
+            tribe.memeServerLocation &&
+            tribe.stakworkApiKey &&
+            tribe.stakworkWebhook
+          ) {
+            const callRecord = (await models.CallRecording.create({
+              recordingId: updatedCallId,
+              chatId: tribe.id,
+              createdBy: message.member.id!,
+              status: constants.call_status.new,
+            })) as CallRecordingRecord
+            let timeActive = 0
+            const interval = setInterval(async function () {
+              timeActive += 60000
+              const file = await fetch(`${tribe.memeServerLocation}`, {
+                method: 'GET',
+                headers: { 'Content-Type': 'application/json' },
+              })
+              // If recording is found
+              if (file.ok) {
+                // Push to stakwork
+                const sendFile = await fetch(
+                  `https://jobs.stakwork.com/api/v1/projects`,
+                  {
+                    method: 'POST',
+                    headers: {
+                      'Content-Type': 'application/json',
+                      Authorization: `Token token="${tribe.stakworkApiKey}"`,
+                    },
+                    body: JSON.stringify({
+                      name: `${updatedCallId} file`,
+                      workflow_id: 3235,
+                      webhook_url: `${tribe.stakworkWebhook}`,
+                      workflow_params: {
+                        media_to_local: {
+                          params: {
+                            media_url: `${tribe.memeServerLocation}`,
+                          },
+                        },
+                      },
+                    }),
+                  }
+                )
+                if (sendFile.ok) {
+                  const res = await sendFile.json()
+                  console.log(res.data)
+                  //update call record to stored
+                  callRecord.update({ status: constants.call_status.stored })
+                  clearInterval(interval)
+                  const embed = new Sphinx.MessageEmbed()
+                    .setAuthor('CallRecordingBot')
+                    .setDescription('Call was recorded successfully')
+                  message.channel.send({ embed })
+                  return
+                } else {
+                  throw `Could not store in stakwork ${sendFile.status}`
+                }
+              }
+              // If recording not found after specified time then it returns an error
+              if (timeActive === 180000 && !file.ok) {
```

Comment:
is this just 3 minutes?

Reply:
This is for test, I just updated it with my recent push